DuckDuckGo/TabViewControllerBrowsingMenuExtension.swift

```diff
@@ -23,6 +23,7 @@ import BrowserServicesKit
 import Bookmarks
 import simd
 import WidgetKit
+import Common
```

Comment:
Do we need this import?

Reply:
Yeah I had a compiler error if I didn't change it. Maybe it got mixed up though.

Reply:
Nah that's okay, happy to leave this in – I've had the same thing happen before. 🙂